bundle/src/main/java/com/adobe/acs/commons/fam/impl/ActionManagerImpl.java

```diff
@@ -144,7 +144,7 @@ public void withResolver(CheckedConsumer<ResourceResolver> action) throws Except
         resolver.setCurrentItem(currentPath.get());
         try {
             action.accept(resolver.getResolver());
-        } catch (Exception ex) {
+        } catch (Throwable ex) {
```

Comment:
If you look at the purpose of the catch logic it is supposed to report ANY cause of failure.  Most Error subclass stuff will likely halt the JVM before it hits this but NOT looking for problems causes other issues down the line when the action manager finishes, hasn't caught fatal issues, and acts like everything is hunky dory.  That is very bad; because without knowing there was a fatal problem somewhere, other code down the line will continue blissfully ignorant of the issue.  So by catching and logging that any Throwable is encountered, this makes for a more seamless and reliable transition from simple one-thread iterative logic into the multi-threaded rapid-fire logic that FAM allows.